assets/settings.ramsalt.local.php

```diff
@@ -87,66 +53,3 @@
 // Drupal access settings.
 $settings['rebuild_access'] = TRUE;
 $settings['skip_permissions_hardening'] = TRUE;
-
-
-// Stage File Proxy settings. Do not change them, they can be configured on top of the file.
-if ($real_host) {
-  $real_host_directory = isset($real_host_directory) ? $real_host_directory : $real_host;
-  if (isset($real_host_access) )
-    $config['stage_file_proxy.settings']['origin'] = ($real_host_use_https ? 'https://' : 'http://').$real_host_access.'@'.$real_host;
-  else
-    $config['stage_file_proxy.settings']['origin'] = ($real_host_use_https ? 'https://' : 'http://').$real_host;
-
-  $config['stage_file_proxy.settings']['hotlink'] = FALSE;
-  $config['stage_file_proxy.settings']['origin_dir'] = 'sites/'.$real_host_directory.'/files';
-}
-
-// Configure the database.
-$default_db_host      = 'mariadb';
-$default_db_port      = '3306';
-$default_db_name      = 'drupal';
-$default_db_user      = 'drupal';
-$default_db_password  = 'drupal';
-
-// Use .env database credential format if provided.
```

Comment:
The removal of the following lines has broken our automated tests. @jonsimonsen @jfauske we need to look into this.

Reply:
@szeidler Can you share an example of a failed job? Quickly looking at the tests job in the orb I don't immediately see where the issue arises,

Reply:
I already pushed a fix.